src/app/services/daysLeftToDeadlineService/days-left-to-deadline.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DaysLeftToDeadlineService {
+  daysLeftToDeadline(dueDate: Date, currentDate: Date): number {
+    const diffInMonths = dueDate.getTime() - currentDate.getTime();
```

Comment:
`diffInMonths` is not a correct variable name here since `getTime` method doesn't return months.